site/README.md

```diff
@@ -49,10 +49,14 @@ To see changes in Javadoc, run:
 
 ### Publishing
 
-After site changes are committed, you can publish the site with this command:
+After site changes are committed, you can publish the site with this series of commands:
 
 ```
-mkdocs gh-deploy
+rm -r site
+mkdir site
+cp ../.asf.yaml site/
```

Comment:
As this uses `../.asf.yaml`, which directory does this document assume we are running these commands from?

Reply:
Good point. I updated the README to include this information.

Reply:
If we're in the `$projectRoot/site` directory, should it be `rm -r docs && mkdir docs`?

My site directory looks as follows.

```
$ ls -la site
total 24
drwxr-xr-x   6 kbendickson  staff   192B Aug 19 16:08 .
drwxr-xr-x  59 kbendickson  staff   1.8K Aug 19 16:08 ..
-rw-r--r--   1 kbendickson  staff   1.6K Aug 19 16:08 README.md
drwxr-xr-x  43 kbendickson  staff   1.3K Aug 20 00:48 docs
-rw-r--r--   1 kbendickson  staff   3.5K Aug 19 16:08 mkdocs.yml
-rw-r--r--   1 kbendickson  staff   872B Aug 19 16:08 requirements.txt
```

If I enter the site directory, and `rm -r site`, I get an exception.

```
$ cd site
$ rm -r site
rm: site: No such file or directory
```

Reply:
I am a bit confused, why is site removed and recreated in this process?

Reply:
@jackye1995, normally the site directory is removed and built entirely from the docs directory when gh-deploy runs. Removing the site directory, copying the file by hand, and using `--dirty` does basically the same thing, but allows us to end up with .asf.yaml in the output that gets copied to the asf-site branch. That's required for publishing now, which is why this change is needed.